crates/pera-core/src/dwallet_mpc/mpc_manager.rs

```diff
@@ -104,13 +127,18 @@ impl DWalletMPCManager {
             weighted_threshold_access_structure,
             weighted_parties,
             batched_sign_sessions: HashMap::new(),
-            outputs_manager: DWalletMPCOutputsManager::new(&epoch_store),
+            outputs_manager: outputs_manager.clone(),
+            status,
         };
+        if let Err(err) = manager.handle_end_of_delivery().await {
+            error!("Failed to advance network DKG first round: {:?}", err);
+        }
```

Comment:
This is redundant as a new checkpoint is being created every 0.5s, and then this function would get called anyway.
```suggestion
```